### PR TITLE
Semantic release setup + README

### PR DESCRIPTION
**Pull request by bb441db**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,53 @@
...
+
+# 📖 Table of contents
+
+1. [Installation](#%EF%B8%8F-installation)
+1. [Basic](packages/eslint-config-basic/README.md)
+1. [Cypress Basic](packages/eslint-config-cypress-basic/README.md)
+1. [Cypress TS](packages/eslint-config-cypress-ts/README.md)
+1. [React](packages/eslint-config-react/README.md)
+1. [TypeScript](packages/eslint-config-ts/README.md)
+1. [Vue](packages/eslint-config-vue/README.md)
```
Comment (thomasowow) [inline README.md:22]: I would put this under a single topic and list all the packages there.
```suggestion
1. [Packages](#packages)
```

File: packages/eslint-config-cypress-basic/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@owowagency/eslint-config-cypress-basic",
-    "version": "0.1.13",
+    "version": "1.0.2",
```
Comment (thomasowow) [inline packages/eslint-config-cypress-basic/package.json:3]: Is this correct?

File: packages/eslint-config-vue/index.js [hunk 3]
```diff
@@ -19,7 +19,6 @@ module.exports = {
     rules: {
         'vue/html-indent': ['error', 4],
         'vue/multi-word-component-names': 'off',
-        'vue/padding-line-between-tags': 'error',
```
Comment (thomasowow) [inline packages/eslint-config-vue/index.js:22]: You sure this should be removed?

**Review by thomasowow — APPROVED** [review]

File: packages/eslint-config-cypress-basic/package.json (hunk 2 above)
Comment (bb441db) [reply packages/eslint-config-cypress-basic/package.json:3]: Yes, starts from 1.0.0 and tested version bumping twice 😀

File: packages/eslint-config-vue/index.js (hunk 3 above)
Comment (bb441db) [reply packages/eslint-config-vue/index.js:22]: Nope

**Review by thomasowow — APPROVED** [review]

**Pull request merged**